doc/classes/ItemList.xml

```diff
@@ -405,6 +412,7 @@
 			Maximum columns the list will have.
 			If greater than zero, the content will be split among the specified columns.
 			A value of zero means unlimited columns, i.e. all items will be put in the same row.
+			[member wraparound_items] takes priority over the max_columns value.
```

Comment:
```suggestion
			[member wraparound_items] takes priority over the [member max_columns] value.
```
Although this note is probably redundant. The properly already says "max columns", which implies that there can be less columns.

Reply:
Would you prefer to remove it? (because it is already a pretty large tooltip)
If not, should I change the wording to e.g. “... over this setting.” because I thought the redundant link to the setting whose description you are reading might lead to the confusion that you think you are at a different setting.

Reply:
Yeah, removing is better IMO.